### PR TITLE
Add eth stake

### PR DESCRIPTION
**Pull request by gokula-krishna-dev**
| Q                        | A 
| ------------------------ | ---
| Fixed Issues?            | 
| Patch: Bug Fix?          | Yes
| Major: Breaking Change?  | No
| Minor: New Feature?      | No
| Tests Added + Pass?      | Yes
| Does not reduce coverage?| Yes
| Any Dependency Changes?  | No

### CODE REVIEW

File: src/botPage/view/View.js [hunk 1]
```diff
@@ -153,38 +154,45 @@ export default class View {
...
-                    ReactDOM.render(<TradeInfoPanel />, $('#summaryPanel')[0]);
-                    ReactDOM.render(<LogTable />, $('#logTable')[0]);
-                    resolve();
+            updateConfigCurrencies().then(() => {
+                symbolPromise.then(() => {
+                    updateTokenList();
+                    this.blockly = new _Blockly();
+                    this.blockly.initPromise.then(() => {
+                        this.setElementActions();
+                        this.constructor.initRealityCheck();
```
Comment (aminmarashi) [inline src/botPage/view/View.js]: Why not simply `this.initRealityCheck`?

File: src/botPage/view/View.js [hunk 2]
```diff
@@ -153,38 +154,45 @@ export default class View {
...
+                        this.constructor.renderReactComponents();
+                        resolve();
+                    });
                 });
             });
         });
     }
+    static initRealityCheck() {
```
Comment (aminmarashi) [inline src/botPage/view/View.js]: Why static? Constructor is able to run these even when they're not static.

Comment (gokula-krishna-dev) [reply src/botPage/view/View.js]: Constructors are exempt from the rule.

File: src/botPage/view/View.js (hunk 1 above)
Comment (gokula-krishna-dev) [reply src/botPage/view/View.js]: this is a way to call static method inside the class.

**Pull request merged**